includes/Autoloader.php

```diff
@@ -13,12 +13,6 @@
  * Dynamically loads the class attempting to be instantiated elsewhere in the
  * plugin by looking at the $class_name parameter being passed as an argument.
  *
- * The argument should be in the form: TutsPlus_Namespace_Demo\Namespace. The
- * function will then break the fully-qualified class name into its pieces and
- * will then build a file to the path based on the namespace.
- *
- * The namespaces in this plugin map to the paths in the directory structure.
- *
  * @param string $class_name The fully-qualified name of the class to load.
  */
 function altapay_namespace_autoload( $class_name ) {
```

Comment:
It should be possible to fully remove this file and just use composer autoload directives instead. If not PSR0/4 then at least should work class-map.